src/main/java/eu/dissco/backend/domain/jsonapi/ObjectType.java

```diff
@@ -0,0 +1,19 @@
+package eu.dissco.backend.domain.jsonapi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum ObjectType {
```

Comment:
Don't see much usage of this enum, only on one line in AnnotationController. What is the use of this enum?

Reply:
Removed